backend/test/resolve-payload-cids.test.js

```
@@ -108,13 +109,61 @@ describe('deal-observer-backend piece indexer payload retrieval', () => {
   let pgPool
   const payloadCid = 'PAYLOAD_CID'
   const minerPeerId = 'MINER_PEER_ID'
-  const pieceCid = 'PIECE_CID'
   const now = Date.now()
   const fetchMinerId = async () => {
     return { PeerId: minerPeerId }
   }
+  /**
+  * @param {Static<typeof ActiveDeal >[]} activeDeals
+  **/
+  const storeActiveDealsCustom = async (activeDeals) => {
```

The adjective "custom" does not tell us much about how this function differs from the version used in production. Let's find a more descriptive name please.

What information we want to convey:
- The production version handles duplicate entries by silently ignoring conflicts
- The version used in tests throws an error when attempting to insert a duplicate deal

Proposed name for the test:

- `storeUniqueActiveDeals`

I like that this name hints at the person reading the code to check whether the deal fields have unique values.

In other places, I prefix test-data builders with `given`. @pyropy uses the prefix `with`. These are viable options, too.

- `givenActiveDeals`
- `givenUniqueActiveDeals`
- `withActiveDeals`
- `withUniqueActiveDeals`

Which name do you prefer? Can you come up with an even better alternative? Let's discuss.

---

I think the name `withUniqueActiveDeals` sounds good and is descriptive. Thanks for the suggestion. 
ae43469